python_modules/dagster/dagster/_core/storage/event_log/in_memory.py

```diff
@@ -92,3 +100,7 @@ def end_watch(self, run_id: str, handler: Callable):
     @property
     def is_persistent(self) -> bool:
         return False
+
+    def dispose(self):
+        self._engine = None
+        self._conn = None
```

Comment:
explicit close? I dont expect this to change any behavior compared to GC on exit